packages/core/src/domain/session/oldCookiesMigration.ts

```diff
@@ -18,13 +17,14 @@ export const LOGS_SESSION_KEY = 'logs'
  * This migration should remain in the codebase as long as older versions are available/live
  * to allow older sdk versions to be upgraded to newer versions without compatibility issues.
  */
-export function tryOldCookiesMigration(options: CookieOptions) {
-  const sessionString = getCookie(SESSION_COOKIE_NAME)
-  const oldSessionId = getCookie(OLD_SESSION_COOKIE_NAME)
-  const oldRumType = getCookie(OLD_RUM_COOKIE_NAME)
-  const oldLogsType = getCookie(OLD_LOGS_COOKIE_NAME)
+export function tryOldCookiesMigration(cookieName: string, cookieStoreStrategy: SessionStoreStrategy) {
```

Comment:
Because of circular dependency.
- I could use `cookieStoreStrategy.retrieveSession`  instead of ` getCookie(cookieName)`, but that is not exactly the same behavior.
- I could move SESSION_COOKIE_NAME outside of sessionInCookie, but it makes sense to keep it there.